3-Authorization-II/2-call-api-b2c/README.md

```diff
@@ -1,193 +1,226 @@
+    1. Select the **Add a permission** button and then:
+    1. Ensure that the **My APIs** tab is selected.
+    1. In the list of APIs, select the API `ms-identity-react-c3s2-api`.
+      * Since this app signs-in users, we will now proceed to select **delegated permissions**, which is requested by apps that signs-in users.
+      * In the **Delegated permissions** section, select **ToDoList.Read**, **ToDoList.ReadWrite** in the list. Use the search box if necessary.
+    1. Select the **Add permissions** button at the bottom.
+    1. At this stage, the permissions are assigned correctly, but since it's a B2C tenant, the users themselves cannot consent to these permissions. To get around this problem, we'd let the [tenant administrator consent on behalf of all users in the tenant](https://docs.microsoft.com/azure/active-directory/develop/v2-admin-consent). Select the **Grant admin consent for {tenant}** button, and then select **Yes** when you are asked if you want to grant consent for the requested permissions for all accounts in the tenant. You need to be a tenant admin to be able to carry out this operation.
+
+##### Configure the client app (ms-identity-react-c3s2-spa) to use your app registration
 
 Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
 
 > In the steps below, "ClientID" is the same as "Application ID" or "AppId".
 
 1. Open the `SPA\src\authConfig.js` file.
-1. Find the key `msalConfig.auth.clientId` and replace the existing value with the application ID (clientId) of `msal-react-spa` app copied from the Azure portal.
-1. Find the key `protectedResources.apiHello.scopes` and replace the existing value with the scope of `msal-react-api` that you have exposed in the previous steps (e.g. `https://fabrikamb2c.onmicrosoft.com/helloapi/demo.read`).
+1. Find the key `msalConfig.auth.clientId` and replace the existing value with the application ID (clientId) of `ms-identity-react-c3s2-spa` app copied from the Azure portal.
+1. Find the key `protectedResources.apiTodoList.scopes` and replace the existing value with the Scopes.
 
 To setup your B2C user-flows, do the following:
 
 1. Find the key `b2cPolicies.names` and populate it with your policy names e.g. `signUpSignIn`.
 1. Find the key `b2cPolicies.authorities` and populate it with your policy authority strings e.g. `https://<your-tenant-name>.b2clogin.com/<your-tenant-name>.onmicrosoft.com/b2c_1_susi`.
 1. Find the key `b2cPolicies.authorityDomain` and populate it with the domain portion of your authority string e.g. `<your-tenant-name>.b2clogin.com`.
 
-## Running the sample
+### Step 4: Running the sample
 
-- Run the service app:
+From your shell or command line, execute the following commands:
 
 ```console
     cd 3-Authorization-II/2-call-api-b2c/API
     npm start
 ```
 
-- In a separate terminal, run the client app:
-
 ```console
-    cd 3-Authorization-II/2-call-api-b2c/SPA
+     cd 3-Authorization-II/2-call-api-b2c/SPA
```

Comment:
>3-Authorization-II/2-call-api-b2c/SPA [](http://example.com/codeflow?start=8&length=37)

get this generated as well via "SampleSubPath" in sample.json

Reply:
done